src/renderer/views/zoom/components/App/style.ts

```diff
@@ -0,0 +1,37 @@
+import styled, { css } from 'styled-components';
+
+import { ITheme } from '~/interfaces';
+import { DialogStyle } from '~/renderer/mixins/dialogs';
+
+export const StyledApp = styled(DialogStyle)`
+  padding: 16px;
+
+  ${({ theme }: { theme?: ITheme; visible: boolean }) => css`
+    color: ${theme['dialog.lightForeground'] ? '#fff' : '#000'};
```

Comment:
I think it isn't needed, since the `DialogStyle` overrides the color.

Reply:
That part was actually (like a lot of code in this commit) lazily copied from the add-bookmark dialog (because it is very similar).

The Add Bookmark dialog has tho following in "src/renderer/views/add-bookmark/components/App/style.ts":
```js
export const StyledApp = styled(DialogStyle)`
  padding: 16px;

  & .textfield,
  .dropdown {
    width: 255px;
    margin-left: auto;
  }

  ${({ theme }: { theme?: ITheme; visible: boolean }) => css`
    color: ${theme['dialog.lightForeground'] ? '#fff' : '#000'};
  `}
`;
```

Reply:
Oh, ok, nevermind